imagersite/imager_images/models.py

```diff
@@ -0,0 +1,95 @@
+"""Create your models here."""
+
+from django.db import models
+from django.contrib.auth.models import User
+
+
+PUBLISHED_STATUS = (
+    ('PB', 'public'),
+    ('PV', 'private'),
+    ('SH', 'shared'),
+)
+
+
+class PhotoManager(models.Manager):
+    """Manage photos."""
+
+    def get_queryset(self):
+        """Return published photos."""
+        return super(PhotoManager, self).get_queryset().filter(published="PB").all()
+
+
+class Photo(models.Model):
+    """Create a photo model."""
+
+    title = models.CharField(max_length=100, default='')
+    description = models.CharField(max_length=255, default='')
+    date_uploaded = models.DateTimeField(auto_now_add=True)
+    date_modified = models.DateTimeField(auto_now=True)
+    date_published = models.DateTimeField(blank=True, null=True)
+    published = models.CharField(
+        max_length=2,
+        choices=PUBLISHED_STATUS,
+        default='PV'
+    )
+    user = models.ForeignKey(
+        User,
+        related_name="photobuild",
+        null=False,
+        on_delete=models.CASCADE
+    )
+    photo = models.ImageField(
+        upload_to='user_images',
+        null=True
+    )
+
+    objects = models.Manager()
+    published_photos = PhotoManager()
+
+    def __repr__(self):
```

Comment:
You need to include a `__str__` method too which helps with the admin display. It can be the same as this.